src/Sarif.Converters/PyLintConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.CodeAnalysis.Sarif.Converters.PyLintObjectModel;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class PyLintConverter : ToolFileConverterBase
+    {
+        public override void Convert(Stream input, IResultLogWriter output, LoggingOptions loggingOptions)
+        {
+            input = input ?? throw new ArgumentNullException(nameof(input));
+            output = output ?? throw new ArgumentNullException(nameof(output));
+
+            PyLintLogList pyLintDefects = PyLintLogReader.ReadLog(input);
+
+            Tool tool = new Tool
+            {
+                Name = "PyLint",
+                FullName = "PyLint Static Code Analysis"
```

Comment:
Is this the "full name" that the tool itself uses? Or did you invent it? If you invented it, remove it (I don't think it adds anything).